certified-connectors/Gratavid/README.md

```diff
@@ -0,0 +1,16 @@
+# Gratavid Connector
+
+## Prerequisites
```

Comment:
@Wtrapp Please add `Description` section here and add a detailed description of your connector which everyone can undersand.